robocorp-code/src/robocorp_code/inspector/inspector_language_server.py

```diff
@@ -20,6 +22,14 @@ def __init__(self):
     def get_locators_json_path(self, directory: str) -> Path:
         return Path(directory) / "locators.json"
 
+    @log_call(log_level=LogLevel.INFO)
```

Comment:
Humm, I think logs are all logged if you use the `debug` log level already (which is quite verbose), so I'm not sure we should add the additional info message for all these methods.

Reply:
Will clean up and move solely to debug mode.